dart/lib/src/sentry_options.dart

```diff
@@ -61,8 +148,53 @@ class SentryOptions {
     Logger logger,
     this.maxBreadcrumbs = 100,
   }) : _logger = logger;
```

Comment:
```suggestion
  }) {
    _logger = logger ?? _logger;
  }
```

Reply:
it should fix the test. Here if no logger is passed `logger` is set to `null`

Reply:
damn I missed this on the ctor, I removed from the ctor, they are no op by default, thanks